monitoring.go

```diff
@@ -30,6 +30,7 @@ const (
 	LoadBalancerCPUUtilizationPercent        = "v1/insights/lbaas/avg_cpu_utilization_percent"
 	LoadBalancerConnectionUtilizationPercent = "v1/insights/lbaas/connection_utilization_percent"
 	LoadBalancerDropletHealth                = "v1/insights/lbaas/droplet_health"
+	LoadBalancerTLSUtilizationPercent        = "tls_connections_per_second_utilization_percent"
```

Comment:
Should this be `v1/insights/lbaas/tls_connections_per_second_utilization_percent`?

Reply:
indeed, bad copy pasta on my part, 🦅  👁️